src/util.ts

```diff
@@ -10,12 +13,16 @@ export function bytesToBase64url(b: Uint8Array): string {
   return u8a.toString(b, 'base64url')
 }
 
+/**
+ * @deprecated Please use base64ToBytes()
+ */
 export function base64urlToBytes(s: string): Uint8Array {
-  return u8a.fromString(s, 'base64url')
+  return base64ToBytes(s)
 }
 
 export function base64ToBytes(s: string): Uint8Array {
-  return u8a.fromString(s, 'base64pad')
+  const inputBase64Url = s.replace(/\+/g, '-').replace(/\//g, '_').replace(/=/g, '')
```

Comment:
Just wondering if regex vs substr `replace` has any efficiency differences. e.g. `s.replace(/\+/g, '-')` vs `s.replace('+', '-')`. Would not have expected regexes here :)

Reply:
I initially tried the non-regex version but it only replaced the first matching character.

Reply:
ah ok, thanks